tools/fill_conf_values.py

```diff
@@ -36,12 +37,76 @@ def hash_filter(string, htype):
     return h.hexdigest()
 
 
+def nginx_brotli_installed():
+    """Check if the nginx brotli module is installed
+
+    Returns
+    -------
+    installed : bool
+        True if the nginx brotli module is installed, False otherwise
+    dynamic : bool
+        True if the module is dynamically loaded, False otherwise
+    modules_dir : str
+        The directory where the nginx modules are located if dynamic loading is used
```

Comment:
Somehow, a NamedTuple feels like it might work well here, but that's total nitpicking and not even clearly an improvement :)

Reply:
hahaha no of course. I'll try that if I have the time.